nextflow.config

```
@@ -202,6 +202,11 @@ params {
 
     // Genotyping
     genotyping_source = 'raw'
+
+    // Sex determination
+    run_sexdeterrmine                                                = true
```

```suggestion
    run_sexdeterrmine                                                = false
```
the default state for all `run_*` parameters should be false